codalab/worker/worker.py

```diff
@@ -90,6 +92,8 @@ def __init__(
         exit_on_exception=False,  # type: bool
         shared_memory_size_gb=1,  # type: int
         preemptible=False,  # type: bool
+        num_threads=10,  # type: int
```

Comment:
Maybe I should make this an environment variable and a command line variable in worker/main.py and inject it in the docker-compose?